content/en/synthetics/browser_tests/test_results.md

```diff
@@ -11,31 +11,57 @@ further_reading:
 - link: "/synthetics/guide/explore-rum-through-synthetics/"
   tag: "Documentation"
   text: "Explore RUM & Session Replay in Synthetics"
+- link: "/synthetics/dashboards/browser_test/"
+  tag: "Documentation"
+  text: "Learn about the Browser Test Performance Dashboard"
 ---
 
 ## Overview
 
-Test results appear after a Synthetic test executes. Browser test results correspond to the test execution at a point in time in a specific location, browser, and device type.
+Test runs appear in a test details page after a Synthetic test executes. [Sample runs](#sample-runs) correlate to the latest passed and failed test executions over a time interval and in a specific number of locations and devices.
```

Comment:
I don't see a sample runs section on this page. Perhaps you meant #sample-results?

Reply:
```suggestion
Test runs appear in a test details page after a Synthetic test executes. [Sample results](#sample-results) correlate to the latest passed and failed test executions over a time interval and in a specific number of locations and devices.
```